src/graphql/queries.js

```diff
@@ -0,0 +1,12 @@
+import { gql } from 'graphql-tag';
+
+const QUERY_USERS = gql`
+query QUERY_USERS {
+  users {
+    username
+    id
+    password
+  }
+}`;
```

Comment:
I think you're missing indentation here, not sure why prettier didn't suggest it for you:
```
const QUERY_USERS = gql`
  query QUERY_USERS {
    users {
      username
      id
      password
    }
  }
`;
```